src/TodoList.jsx

```diff
@@ -3,35 +3,46 @@ import './App.css'
 
 
 function TodoLIst() {
+  const [todo, setTodo] = useState([]);
+const clickhandler=()=>{
+ //alert(todo);
+}
```

Comment:
Make sure the click handle performs exactly what it is intended to instead of a comment. Otherwise remove it from the code base as it is redundant for now